run.sh

```diff
@@ -68,6 +68,33 @@ else
   echo "[info] Permissions already set for volume mappings" | ts '%Y-%m-%d %H:%M:%.S'
 fi
 
+# Check new certificate and import it to keystore
+pushd /var/lib/unifi-video >/dev/null
+if [ -f "cert/cert.pem" -a -f "cert/privkey.pem" ]; then
+    # convert cert & key
+    mkdir certificates
+    openssl x509 -outform DER -in cert/cert.pem -out certificates/ufv-server.cert.der
```

Comment:
Maybe wrap this in some exit status checking? It'd suck to nuke the certs after the whole thing bombs.

Reply:
And it'd be good to have a little echo spew, like the permissions fix. Something simple to give an idea of what is going on.